atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/qos/ThriftObjectSizeUtils.java

```diff
@@ -37,13 +38,33 @@
 
 public final class ThriftObjectSizeUtils {
 
-    private static final int ONE_BYTE = 1;
+    private static final long ONE_BYTE = 1;
 
     private ThriftObjectSizeUtils() {
         // utility class
     }
 
-    public static int getColumnOrSuperColumnSize(ColumnOrSuperColumn columnOrSuperColumn) {
+    public static long getApproximateWriteByteCount(Map<ByteBuffer, Map<String, List<Mutation>>> batchMutateMap) {
+        long approxBytesForKeys = getCollectionSize(batchMutateMap.keySet(), ThriftObjectSizeUtils::getByteBufferSize);
+        long approxBytesForValues = getCollectionSize(batchMutateMap.values(), currentMap ->
+                getCollectionSize(currentMap.keySet(), ThriftObjectSizeUtils::getStringSize)
+                        + getCollectionSize(currentMap.values(),
+                        mutations -> getCollectionSize(mutations, ThriftObjectSizeUtils::getMutationSize)));
+        return approxBytesForKeys + approxBytesForValues;
+    }
+
+    public static long getApproximateReadByteCount(Map<ByteBuffer, List<ColumnOrSuperColumn>> result) {
+        return getCollectionSize(result.entrySet(),
+                rowResult -> ThriftObjectSizeUtils.getByteBufferSize(rowResult.getKey())
```

Comment:
We don't need to specify `ThriftObjectSizeUtils.` or `ThriftObjectSizeUtils::` (throughout this class)